Homework patterns.md

```diff
@@ -0,0 +1,14 @@
+Design Patterns: Is a solution to fix the common problem than can occur in software design. The design pattern can considered as a templates because aren’t complete and this allow that we can used them by different ways to solve problems.
```

Comment:
Please can you re-write this definition of Design Patterns? I didn't understand what are you talking about.